groups/instance/variables.tf

```diff
@@ -4,12 +4,24 @@
 
 variable "account_name" {
   description = "The name of the AWS account we are using"
+  default     = "shared-services"
+  type        = string
+}
+
+variable "secrets_account_name" {
```

Comment:
Not needed

Reply:
Implemented via commit: 252d107678658864551f07a8c8958fea666199bc